src/kernels/MIOpenBatchNormActivBwdPerAct.cl

```diff
@@ -33,8 +33,10 @@
 #pragma clang diagnostic ignored "-Wsometimes-uninitialized"
 #endif
 
-#if(MIO_BN_GFX1030 == 1 && __AMDGCN__)
-#undef __AMDGCN__
+#define MIOPEN_USE_AMDGCN 0
+#if __AMDGCN__ && MIO_BN_GFX1030 != 1
```

Comment:
Sorry, my recommendation was a bit wrong. `#if __AMDGCN__ && ...` would lead to at least build warning when `__AMDGCN__` is not defined. Should be:
```suggestion
#if defined(__AMDGCN__) && MIO_BN_GFX1030 != 1
```